--- a/auto/feature
+++ b/auto/feature
@@ -49,6 +49,12 @@ eval "/bin/sh -c \"$ngx_test\" >> $NGX_AUTOCONF_ERR 2>&1"
 
 if [ -x $NGX_AUTOTEST ]; then
 
+    # cross-built autotest cannot be run on host platform
+    # just assume "found" if autotest can be built with the feature
+    if [ "$NGX_SYSTEM" = Darwin ]; then
+        ngx_feature_run=no 
+    fi
+
     case "$ngx_feature_run" in
 
         yes)