--- a/src/os/unix/ngx_darwin_config.h
+++ b/src/os/unix/ngx_darwin_config.h
@@ -98,3 +98,15 @@ extern char **environ;
 
 
 #endif /* _NGX_DARWIN_CONFIG_H_INCLUDED_ */
+
+#define NGX_PTR_SIZE            8
+#define NGX_SIZE_T_LEN          (sizeof("-9223372036854775808") - 1)
+#define NGX_MAX_SIZE_T_VALUE    9223372036854775807LL
+#define NGX_TIME_T_LEN          (sizeof("-9223372036854775808") - 1)
+#define NGX_TIME_T_SIZE         8
+#define NGX_MAX_TIME_T_VALUE    9223372036854775807LL
+#define NGX_OFF_T_LEN           (sizeof("-9223372036854775808") - 1)
+#define NGX_MAX_OFF_T_VALUE     9223372036854775807LL
+#define NGX_SIG_ATOMIC_T_SIZE   4
+
+#define NGX_HAVE_LITTLE_ENDIAN  1
\ No newline at end of file
