diff --git a/auto/feature b/auto/feature
index 3561f59..06b087d 100644
--- a/auto/feature
+++ b/auto/feature
@@ -62,6 +62,11 @@ if [ -x $NGX_AUTOTEST ]; then
                 fi
 
             else
+                echo "cannot execute"
+                ngx_found=yes
+                echo "HERE ERR1"
+                cat $NGX_AUTOCONF_ERR
+                echo "END1"
                 echo " found but is not working"
             fi
         ;;
@@ -80,6 +85,11 @@ if [ -x $NGX_AUTOTEST ]; then
 
 END
             else
+                echo "cannot execute"
+                ngx_found=yes
+                echo "HERE ERR2"
+                cat $NGX_AUTOCONF_ERR
+                echo "END2"
                 echo " found but is not working"
             fi
         ;;
