diff --git a/auto/feature b/auto/feature
index 3561f5941..c26e656c2 100644
--- a/auto/feature
+++ b/auto/feature
@@ -62,6 +62,9 @@ if [ -x $NGX_AUTOTEST ]; then
                 fi
 
             else
+                echo "HERE ERR"
+                cat $NGX_AUTOCONF_ERR
+                echo "END"
                 echo " found but is not working"
             fi
         ;;
@@ -80,6 +83,9 @@ if [ -x $NGX_AUTOTEST ]; then
 
 END
             else
+                echo "HERE ERR"
+                cat $NGX_AUTOCONF_ERR
+                echo "END"
                 echo " found but is not working"
             fi
         ;;
